buildconfig.env.example

```diff
@@ -1,3 +0,0 @@
-export sauceport=5555
```

Comment:
This file exists as an example for SauceLabs frontend unit testing.
I'm happy to remove it though. Not used for 1 year now.